server/containers/rpmrepo/processincoming.sh

```diff
@@ -49,7 +49,8 @@ while read -r -u 10 pkgname; do
     fi
 
     mkdir -p /tmp/repo-staging/
-    mv "${pkg}" /tmp/repo-staging
+    cp "${pkg}" /tmp/repo-staging
+    rm -f "${pkg}"
```

Comment:
This is necessary because we later want to modify this file and that raised a permission error since the processor's matching group only had reading permissions on the original file.